ale/drivers/base.py

```diff
@@ -896,9 +906,9 @@ def _init_tables(self):
             elif table['Name'] == 'BodyRotation':
                 self._body_orientation_table = parse_rotation_table(table, field_data)
             elif table['Name'] == 'InstrumentPosition':
-                self._inst_position_table = parse_position_table(field_data)
+                self._inst_position_table = parse_position_table(table, field_data)
```

Comment:
a little tangential to this PR perhaps, but why does this method include this if/else? Why not just push the parse into the method that dispatches here?

Reply:
I'm not sure what you mean by the method that dispatches here? This method parses the label and pulls off multiple tables. There's nothing to do the if/else on outside of this method.

Reply:
I am advocating getting rid of this method entirely. The if/else is at best going to be a pain to test. Why not just inline the parse_position_table into the self._inst_position_table for example? More on that in the comment below then.

Reply:
I am advocating getting rid of this method entirely. The if/else is at best going to be a pain to test. Why not just inline the parse_position_table into the self._inst_position_table for example? More on that in the comment below then.